src/PropertySuggester/Suggesters/SimpleSuggester.php

```diff
@@ -172,14 +172,15 @@ public function suggestByItem( Item $item, $limit, $minProbability, $context ) {
 
 				$entityId = $dataValue->getEntityId();
 
-				if ( !( $entityId instanceof Int32EntityId ) ) {
+				if ( !( $entityId instanceof ItemId ) ) {
 					throw new LogicException(
-						$entityId->getSerialization() . ' expected to be Int32EntityId'
+						"Property $numericPropertyId in wgPropertySuggesterClassifyingPropertyIds"
+						. ' does not have property type wikibase-item'
```

Comment:
Not sure what you mean. The value type is `EntityIdValue`. This is checked above. The line we are talking about here checks if the referenced entity is an item. This is a property of the properties data type (a.k.a. "property type").